css/modules.css

```diff
@@ -0,0 +1,62 @@
+/* Desktop version */
+#a{
+    background-color: red;
+}
+#b{
+    background-color: blue;
+}
+#c{
+    background-color: green;
+}
+#d{
+    background-color: yellow;
+}
+#e{
+    background-color: grey;
+}
+#f{
+    background-color: purple;
+}
+#g{
+    background-color: orange;
+}
+#h{
+    background-color: pink;
+}
+#i{
+    background-color: cyan;
+}
+
+
+
+@media(max-width: 480px){
```

Comment:
As a mobile-first approach, it is better to not depend on the media query for the CSS rules we want for mobile view. Let your mobile view rules be the default. Then you only need to use the media query for other layouts.